vhd/packer/configure-windows-vhd.ps1

```diff
@@ -85,9 +85,9 @@ function Get-FilesToCacheOnVHD
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.1/windowszip/v1.18.1-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://kubernetesartifacts.azureedge.net/azure-cni/v1.0.28/binaries/azure-vnet-cni-windows-amd64-v1.0.28.zip",
             "https://kubernetesartifacts.azureedge.net/azure-cni/v1.0.29/binaries/azure-vnet-cni-windows-amd64-v1.0.29.zip",
-            "https://kubernetesartifacts.azureedge.net/azure-cni/v1.0.30/binaries/azure-vnet-cni-windows-amd64-v1.0.30.zip"
+            "https://kubernetesartifacts.azureedge.net/azure-cni/v1.0.30/binaries/azure-vnet-cni-windows-amd64-v1.0.30.zip",
+            "https://kubernetesartifacts.azureedge.net/azure-cni/v1.0.33/binaries/azure-vnet-cni-windows-amd64-v1.0.33.zip"

```

Comment:
Note this is not valid aks egress

Reply:
wget https://acs-mirror.azureedge.net/cni/azure-vnet-cni-windows-amd64-v1.0.30.zip
--2020-04-22 18:39:10--  https://acs-mirror.azureedge.net/cni/azure-vnet-cni-windows-amd64-v1.0.30.zip
Resolving acs-mirror.azureedge.net (acs-mirror.azureedge.net)... 2600:1405:a000::172d:2e83, 2600:1405:a000::172d:2e60, 23.35.68.224, ...
Connecting to acs-mirror.azureedge.net (acs-mirror.azureedge.net)|2600:1405:a000::172d:2e83|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 7617904 (7.3M) [application/x-zip-compressed]
Saving to: ‘azure-vnet-cni-windows-amd64-v1.0.30.zip’

azure-vnet-cni-windows-amd64-v1.0.30.zip             100%[=====================================================================================================================>]   7.26M  14.9MB/s    in 0.5s    

2020-04-22 18:39:11 (14.9 MB/s) - ‘azure-vnet-cni-windows-amd64-v1.0.30.zip’ saved [7617904/7617904]